test/rewrite/test_transversal.py

```diff
@@ -83,59 +83,6 @@ def test_simple_rewrite(
     assert result.has_done_something is has_done_something
 
 
-def test_get_measurement_result():
```

Comment:
Why these tests are completely gone? If we plan to readd in future we can also use `@pytest.mark.skip`.